karma.conf.js

```diff
@@ -14,6 +14,9 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
+      'jujugui/static/gui/build/app/assets/javascripts/jujulib/juju.js',
```

Comment:
these two lines can be shortened to `'jujugui/static/gui/build/app/assets/javascripts/jujulib/*.js'`